cinema/views.py

```diff
@@ -113,17 +164,26 @@ def get_serializer_class(self):
         return MovieSessionSerializer
 
 
+# -----OrderViewSets-----------------------------------------------------------
+
+
 class OrderPagination(PageNumberPagination):
     page_size = 10
     max_page_size = 100
 
 
-class OrderViewSet(viewsets.ModelViewSet):
+class OrderViewSet(
+    mixins.ListModelMixin,
+    mixins.CreateModelMixin,
+    viewsets.GenericViewSet,
+):
     queryset = Order.objects.prefetch_related(
         "tickets__movie_session__movie", "tickets__movie_session__cinema_hall"
     )
     serializer_class = OrderSerializer
     pagination_class = OrderPagination
+    authentication_classes = (TokenAuthentication,)
+    permission_classes = (IsAuthenticated,)
```

Comment:
The `OrderViewSet` uses `IsAuthenticated` for `permission_classes`, while other viewsets use `IsAdminOrIfAuthenticatedReadOnly`. Ensure this difference is intentional and fits the application's access control requirements.